aazztech/directorist
# fixed listing slider issue

## PR Type
What kind of change does this PR introduce?


## Description
How to reproduce the issue or how to test the changes

1. Create a listing with images.
2. Visit the single listing page of the listing.
3. Check if the slider images are shown or not.

## Issue
[Fixes](https://tasks.hubstaff.com/app/organizations/37274/projects/271564/tasks/6164836)

## Checklist
